apps/web/src/components/commands/nft/Nft.tsx

```diff
@@ -1,12 +1,19 @@
 import { DataPill } from "../../DataPill";
 import { useQuery } from "react-query";
 import Image from "next/image";
-import { NftPortAPIAssetResponse, NftPortAPIStatsResponse } from "./types";
+import {
+  SimpleHashNFTDetailsResponse,
+  SimpleHashCollectionActivityResponse,
+  SimpleHashCollectionResponse,
+} from "./types";
+import { formatEther } from "../../../utils/format";
+import { exponentialToDecimal } from "../../../utils/exponents";
 
 export type NftWidgetProps = {
   property: "holders" | "floor" | "total-sales-volume" | "image";
   address: string;
-  chain: number;
+  chainName: string;
```

Comment:
To reduce the amount of props we need to handle, could we find this based on the chainId in the component instead? Something like
```ts
const chainName = SUPPORTED_CHAINS.find(c => c.id === chainId)?.name
```
Then we just have one single source of truth!

Reply:
> 

Alright! done now!